implementation/choreography-to-orchestration/lib/orchestration-stack.ts

```diff
@@ -0,0 +1,214 @@
+import { Construct } from "constructs";
+import {
+  Stack,
+  StackProps,
+  aws_lambda as lambda,
+  aws_apigateway as apigateway,
+  aws_iam as iam,
+  aws_dynamodb as dynamodb,
+  aws_sns as sns,
+  aws_sqs as sqs,
+  aws_lambda_destinations as destinations,
+  aws_lambda_event_sources as sources,
+  aws_stepfunctions as sfn,
+  aws_stepfunctions_tasks as tasks,
+  CfnOutput,
+} from "aws-cdk-lib";
+import { NagSuppressions } from 'cdk-nag'
+
+
+export class OrchestrationStack extends Stack {
```

Comment:
Looks like your implementation does not have the dynamodb which is shown in your pattern  `patterns/images/choreography-to-orchestration/orchestration.png.`

Remember, we want to ensure that any reader is able to understand the difference in the original and refactored. They will be confused as to why dynamodb got removed in the refactored architecture.

Reply:
Added Dynamodb table